lib/suma/automation_trigger/funding_transaction_match.rb

```diff
@@ -41,7 +42,7 @@ def run
         originating_ledger: Suma::Payment::Account.lookup_platform_vendor_service_category_ledger(vsc),
         receiving_ledger: ledger,
         associated_vendor_service_category: vsc,
-        memo: Suma::TranslatedText.create(**params.fetch(:subsidy_memo)),
+        memo: contribution_text,
```

Comment:
Allow memo to be optional, and default to the contribution text. There will be cases where we want a separate memo.